### DISCUSSION TITLE
Handsfree Snooze

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2124] [Ideas]
While driving a car, the safest approach when hearing an alert is to stop driving.

There are times you may get a high alert that you expected.  For example, you have already taken insulin with your meal and you expect that you will get a temporary high, which will drop shortly after because of the insulin you have already taken.  But, what do you do when the alert goes off?  Do you stop the car?  That's the safest approach of course.
Do you attempt to snooze while driving the car?  That's very dangerous and in some areas even illegal.

What are the existing features that can be used?

Do we need to add a feature?

@tolot27 @jamorham @tzachi-dar @psonnera @joshbohmer

**Comment by tzachi-dar** [comment]
You can use the volume buttons to stop an alert.
We might also add shake to stop alert (only in the car).

**Reply by Navid200** [reply]
Unfortunately, you need to first unlock the phone.
And it's the unlocking the phone that can be dangerous while driving.
